### ISSUE TITLE
Problem with multiple carriage return on a multi line string

### ISSUE THREAD

**Issue by Lilja** [yoavniran/cypress-intercept-formdata#6]
Hi! I have the following form data:
```
'\r\nContent-Disposition: form-data; name="TICKET.content"\r\n\r\n\r\nFunnel instance: global\r\nSubscription URL: <link>\r\nAccount URL: <link2>/\r\nDo this and that\r\n--'
```

The problem with this is because of the else-case in `getFormDataFromRequest `, it splits the string with 
```JavaScript
p.split(`\r\n`)
```
and magically picks the third index. The third index of this returns an empty string:

```
p.split(`\r\n`)
```
![image](https://user-images.githubusercontent.com/6134511/155317990-ed434e2d-9482-4692-be4b-a21a6a753515.png)

I wonder if there are other ways other than splitting the string with `\r\n` to get that text blob.

**Comment by yoavniran** [comment]
Hi @Lilja I will look into this and see what I can do.
Not sure why you're seeing this problem while I havent encountered it yet.
Can you share your test (here or link to repo)?

**Comment by yoavniran** [comment]
@Lilja should be fixed now with 0.4.0.
Plz let me know if there are still issues or solved for you

**Comment by Lilja** [comment]
Awesome, thanks for the quick resolution.
